core/config/v2/docs/chains-evm.toml

```diff
@@ -309,3 +309,7 @@ WSURL = 'wss://web.socket/test' # Example
 HTTPURL = 'https://foo.web' # Example
 # SendOnly limits usage to sending transaction broadcasts only. With this enabled, only HTTPURL is required, and WSURL is not used.
 SendOnly = false # Default
+
+[EVM.Automation]
+# TransmitGasLimit controls the gas limit for transmit transactions from ocr2automation job.
+TransmitGasLimit = 5300000 # Default
```

Comment:
cc @jmank88 I want to add a chain specific config for keepers v2 (now known as automation). 
Since this is chain specific the only place I could find for this was under EVM, but not sure if this is the right place